src/sentry/static/sentry/app/actionCreators/modal.jsx

```diff
@@ -76,3 +76,13 @@ export function openCommandPalette(options = {}) {
       });
     });
 }
+
+export function openRecoveryOptions(options = {}) {
+  import(/* webpackChunkName: "RecoveryOptionsModal" */ '../components/modals/recoveryOptionsModal')
```

Comment:
I know the rest of the file doesn't do this (because our mass refactor didn't catch these), but can we use absolute paths here? e.g. `app/components/modals...`